### PR TITLE
[New Rule] Group Policy Abuse for Privilege Addition

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Solves #1602 

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/privilege_escalation_group_policy_privileged_groups.toml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+for example: SeTakeOwnershipPrivilege, SeEnableDelegationPrivilege, etc.
+- Inspect the user SIDs associated with these privileges
+
+### False Positive Analysis
+- Verify if these User SIDs should have these privileges enabled.
+- Inspect whether the user that has done these modifications should be allowed to do it. The user name can be found in the
+winlog.event_data.SubjectUserName field
+
+### Related Rules
+-
```
Comment (w0rk3r) [inline rules/windows/privilege_escalation_group_policy_privileged_groups.toml:38]: I'm going to submit 2 related rules, will update this one once done.

Comment (w0rk3r) [reply rules/windows/privilege_escalation_group_policy_privileged_groups.toml]: Done

**Review by Samirbous — APPROVED** [review]

File: rules/windows/privilege_escalation_group_policy_privileged_groups.toml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- This attack abuses a legitimate mechanism of the Active Directory, so it is important to determine whether the activity is legitimate
+and the administrator is authorized to perform this operation.
+- Retrieve the contents of the `GptTmpl.inf` file, under the `Privilege Rights` section, look for potentially dangerous high privileges,
+for example: SeTakeOwnershipPrivilege, SeEnableDelegationPrivilege, etc.
+- Inspect the user SIDs associated with these privileges
+
+### False Positive Analysis
+- Verify if these User SIDs should have these privileges enabled.
+- Inspect whether the user that has done these modifications should be allowed to do it. The user name can be found in the
+winlog.event_data.SubjectUserName field
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_group_policy_privileged_groups.toml:35]:
```suggestion
`winlog.event_data.SubjectUserName` field
```

File: rules/windows/privilege_escalation_group_policy_privileged_groups.toml [hunk 3]
```diff
@@ -0,0 +1,93 @@
+[metadata]
+creation_date = "2021/11/08"
+maturity = "production"
+updated_date = "2021/11/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the modification of Group Policy Object Attributes (for the first time) to add privileges to user accounts or use them to
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_group_policy_privileged_groups.toml:9]: is this a more accurate way to say it

```suggestion
This rule detects the first occurrence of a modification to Group Policy Object Attributes to add privileges to user accounts or use them to
```

**Review by brokensound77 — APPROVED** [review]
Left a question then LGTM 👍

File: rules/windows/privilege_escalation_group_policy_privileged_groups.toml (hunk 3 above)
Comment (w0rk3r) [reply rules/windows/privilege_escalation_group_policy_privileged_groups.toml:9]: Yeah, sounds better

**Pull request merged**